addons/resource/models/resource_resource.py

```diff
@@ -49,6 +54,12 @@ def default_get(self, fields):
         ('check_time_efficiency', 'CHECK(time_efficiency>0)', 'Time efficiency must be strictly positive'),
     ]
 
+    @api.depends('user_id')
+    def _compute_avatar_128(self):
+        for resource in self:
+            user = resource.with_context(active_test=False).user_id
+            resource.avatar_128 = user and user[0].avatar_128
```

Comment:
> `active_test` is indeed, not needed here. 🙂

Then it can just be a related field, right? Does it make sense to have a related field in this module and change it to a computed (non-stored) field in hr? Or is it better to keep a compute method in this module that we override in hr?